### PR TITLE
Refactor Rocksmq's config

### PR DESCRIPTION
**Pull request by jiaoew1991**
Signed-off-by: Enwei Jiao <enwei.jiao@zilliz.com>
issue: #18300

### CODE REVIEW

**Comment by jiaoew1991** [comment]
rerun ut

File: internal/config/env_source.go [hunk 1]
```diff
@@ -76,6 +76,10 @@ func (es EnvSource) GetSourceName() string {
 	return "EnvironmentSource"
 }
 
+func (es EnvSource) SetEventHandler(eh EventHandler) {
+
+}
```
Comment (congqixia) [inline internal/config/env_source.go:81]: Is it ok to leave this empty?

Comment (jiaoew1991) [reply internal/config/env_source.go:81]: Yes, env_source cannot trigger event

Comment (jiaoew1991) [reply internal/config/env_source.go:81]: Yes, env_source cannot trigger event

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**